package.json

```diff
@@ -37,11 +37,11 @@
     "pify": "^2.2.0",
     "rimraf": "^2.4.3",
     "run-applescript": "^2.0.0",
-    "xdg-trashdir": "^2.0.0"
+    "xdg-trashdir": "^2.1.0",
+    "path-exists": "^2.0.0"
```

Comment:
Move this above `pify`. `npm install path-exists --save` adds it to `package.json` automatically in alphabetic order.